robottelo/vm.py

```diff
@@ -172,6 +172,8 @@ def create(self):
         if result.return_code != 0:
             raise VirtualMachineError(
                 u'Failed to run snap-guest: {0}'.format(result.stderr))
+        else:
+            self._created = True
```

Comment:
the `_created` attribute should be set immediately after we manage to run `snap-guest` successfully.